### PR TITLE
Adding docs for popcorn methods, and some small updates.

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

File: popcorn-methods/_posts/2014-01-11-update.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Updates popcorn track event properties on existing track events, like the start time or end time. This can be done dynamically while the video is playing.
+
+## Options ##
+
+* **id** \[Integer\] - id assigned to the track event you wish to update
+* **options** \[object\] - new options object you wish to apply to the existing track event, existing values on the track event that are not in the update options object are left unchanged.
+
+## Use Cases ##
+
+You want to change a track events data on the fly, instead of removing and adding a new modified version of the trck event.
```
Comment (mjschranz) [inline popcorn-methods/_posts/2014-01-11-update.md]: `of the trackevent`

File: popcorn-methods/_posts/2014-01-11-add.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    pop.example({ start: 1 });
+    pop.example("example-id", { start: 1 });
+    pop.example({ start: 1, id: "another-id" });
+
+{% endhighlight %}
+
+## Related ##
+
+* [subtitle](/popcorn-docs/plugins/#subtitle)
+* [image](/popcorn-docs/plugins/#image)
```
Comment (mjschranz) [inline popcorn-methods/_posts/2014-01-11-add.md:42]: Curious. Why is image related?

File: popcorn-methods/_posts/2014-01-11-removeTrackEvent.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+`popcorn.removeTrackEvent()` removes an existing track event from the popcorn instance, via a track event's id.
+
+## Options ##
+
+* **id** \[String\] - id assigned to the track event you wish to remove
+
+## Use Cases ##
+
+You want to remove a track event from the poporn instance
```
Comment (mjschranz) [inline popcorn-methods/_posts/2014-01-11-removeTrackEvent.md]: Missing period.

File: popcorn-methods/_posts/2014-01-11-removeTrackEvent.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+---
+# popcorn.removeTrackEvent( id ) #
+
+## Purpose ##
+
+`popcorn.removeTrackEvent()` removes an existing track event from the popcorn instance, via a track event's id.
+
+## Options ##
+
+* **id** \[String\] - id assigned to the track event you wish to remove
```
Comment (mjschranz) [inline popcorn-methods/_posts/2014-01-11-removeTrackEvent.md]: Missing period.

File: popcorn-methods/_posts/2014-01-11-getTrackEvents.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+---
+title: getTrackEvents
+---
+# popcorn.getTrackEvents() #
+
+## Purpose ##
+
+`popcorn.getTrackEvents()` returns an array track event objects attached to the popcorn instance
```
Comment (mjschranz) [inline popcorn-methods/_posts/2014-01-11-getTrackEvents.md]: Missing period.

File: popcorn-methods/_posts/2014-01-11-getTrackEvent.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+`popcorn.getTrackEvent()` returns the track event assigned to the id parameter
+
+## Options ##
+
+* **id** \[String\] - the id assigned to the desired track event
+
+## Use Cases ##
+
+You want to check values stored in a track event, and have the id
```
Comment (mjschranz) [inline popcorn-methods/_posts/2014-01-11-getTrackEvent.md]: Missing period.

File: popcorn-methods/_posts/2014-01-11-getTrackEvent.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+---
+# popcorn.getTrackEvent( id ) #
+
+## Purpose ##
+
+`popcorn.getTrackEvent()` returns the track event assigned to the id parameter
+
+## Options ##
+
+* **id** \[String\] - the id assigned to the desired track event
```
Comment (mjschranz) [inline popcorn-methods/_posts/2014-01-11-getTrackEvent.md]: Missing period.

File: popcorn-methods/_posts/2014-01-11-getTrackEvent.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
+---
+title: getTrackEvent
+---
+# popcorn.getTrackEvent( id ) #
+
+## Purpose ##
+
+`popcorn.getTrackEvent()` returns the track event assigned to the id parameter
```
Comment (mjschranz) [inline popcorn-methods/_posts/2014-01-11-getTrackEvent.md]: Missing period.

File: popcorn-methods/_posts/2014-01-11-add.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+## Options ##
+
+* **id** \[Integer\] - optional id assigned to your new track event, if omitted a random id is assigned to your track event.
+* **options** \[object\] - object of properties to create a track event from, like start, end or text.
+* **options.id** \[Integer\] - optional id assigned to your new track event, if omitted a random id is assigned to your track event.
+
+## Use Cases ##
+
+You want to add some text or images to your video.
```
Comment (mjschranz) [inline popcorn-methods/_posts/2014-01-11-add.md]: This feels like it should be more fleshed out. I'm unsure how in-depth you want to go really but I think this should have a little more detail.

File: popcorn-methods/_posts/2014-01-11-add.md (hunk 2 above)
Comment (ScottDowne) [inline popcorn-methods/_posts/2014-01-11-add.md:42]: Just two examples of plugins that came to mind.

File: popcorn-methods/_posts/2014-01-11-add.md (hunk 9 above)
Comment (ScottDowne) [inline popcorn-methods/_posts/2014-01-11-add.md]: An example of some sort maybe? Hm.

File: popcorn-methods/_posts/2014-01-11-add.md (hunk 2 above)
Comment (ScottDowne) [inline popcorn-methods/_posts/2014-01-11-add.md:42]: Ah, I remember. I referenced both subtitle and image in the use case, as an example, so I figured I would link to it. My update just references subtitle as an example, and text is the related, image can be removed.

Comment (ScottDowne) [inline popcorn-methods/_posts/2014-01-11-add.md:42]: Wait, no, image is back.

**Pull request merged**